azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/configuration/SupportedLanguages.java

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.configuration;
+
+public enum SupportedLanguages {
```

Comment:
Can you add, isRtl default value to enum, and update the settings UI to automatically select based on it?